frontend/src/metabase/static-viz/components/SmartScalar/SmartScalar.tsx

```diff
@@ -23,12 +23,12 @@ export function SmartScalar({
 
   const settings = computeSmartScalarSettings(rawSeries, dashcardSettings);
 
-  const trend = computeTrend(rawSeries, insights, settings, {
+  const { trend, error } = computeTrend(rawSeries, insights, settings, {
     formatValue,
     getColor,
   });
 
-  if (!trend) {
+  if (error || !trend) {
```

Comment:
It would be great to show a bit more meaningful error on static visualizations too